src/sentry/tagstore/v2/backend.py

```diff
@@ -952,55 +952,6 @@ def get_group_tag_values_for_users(self, event_users, limit=100):
             )
         )
 
-    def get_group_ids_for_search_filter(
```

Comment:
This will conflict with #12212 but the conflict should be easily to solve on either side.